ext-src/commands/DiscardSuggestion.ts

```diff
@@ -14,82 +13,71 @@ export function activateDiscardCommand(context: vscode.ExtensionContext): void {
   const commandHandler = async (args: DiscardCommandHandler) => {
     const currentWorkSpaceFolder = Utils.getRootFolderName();
```

Comment:
Somewhere in here we should call the highlight API once again to remove highlighting on discarded problems